_data/changelogs/component-pagination.yml

```diff
@@ -2,6 +2,15 @@ title: Pagination
 type: component
 changelogURL:
 items:
+- date: NNNN-NN-NN
+  summary: Updated text-decoration style on pagination links so page numbers are underlined.
+  summaryAdditional: Users should confirm the pagination page numbers are underlined.
```

Comment:
**suggestion:** My gut reaction is to use `summaryAdditional` to add context to _why_ we made this change. 

I'm unsure of the _best_ wording here but maybe something like:

```suggestion
  summaryAdditional: Link styles are now visually consistent with usa-link.
```

CC: @amyleadem since you're good at these

Reply:
Resolved in [009e774](https://github.com/uswds/uswds-site/pull/2793/commits/009e774d0d925b3f040a32bc9888e27fb2394fc6).

@amyleadem @mahoneycm - I just added Charlie's recommended summaryAdditional entry but let me know if we want to change it to something else!